website/docs/r/iam_trusted_profiles_link.html.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_trusted_profiles_link"
+description: |-
+  Manages iam_trusted_profiles_link.
+subcategory: "IAM Identity Services"
+---
+
+# ibm_iam_trusted_profiles_link
+
+Provides a resource for iam_trusted_profiles_link. This allows iam_trusted_profiles_link to be created, updated and deleted.
```

Comment:
Create, update, and delete an IAM trusted profiles link. For more information, about IAM trusted profiles, see [](). 

reference link would help here.